src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/DevAppServerArgs.java

```diff
@@ -24,30 +24,42 @@
 public class DevAppServerArgs {
 
   /**
-   * @return [--name=value] or [] if value=null.
+   * @return {@code [--name=value]} or {@code []} if value=null.
    */
   public static List<String> get(String name, String value) {
     return Args.stringWithEq(name, value);
   }
 
   /**
-   * @return [--name=value1, --name=value2, ...] or [] if value=null.
+   * @return {@code [--name=value1, --name=value2, ...]} or {@code []} if value=null.
    */
   public static List<String> get(String name, List<String> values) {
     return Args.stringsWithEq(name, values);
   }
 
   /**
-   * @return [--name=value] or [] if value=null.
+   * @return {@code [--name=value]} or {@code []} if value=null.
    */
   public static List<String> get(String name, Integer value) {
     return Args.integerWithEq(name, value);
   }
 
   /**
-   * @return [--name] if value=true, [] if value=false/null.
+   * @return {@code [--name]} if value=true, {@code []} if value=false/null.
    */
   public static List<String> get(String name, Boolean value) {
+    return get(name, value, false);
+  }
+
+  /**
+   * @return if produceTrueFalse=true, {@code [--name=true]} if value=true, {@code [--name=false]}
```

Comment:
@elharo I don't know how to be clearer here :(. We're adding library support for boolean flags of the kind `--flag=false `.

Perhaps if you tell me if anything specific is generating that confusion, I can be more helpful.

@patflynn as you mentioned, this change is indeed an API change, which goes way beyond the scope of what this PR was supposed to be. Creating a new issue doesn't mean the issue can't be worked on now -- in practice, the same thing will happen in both cases, but without this code committed, and the dev_appserver reload fix will have to wait for that work to be finished in consequence. As you wish, though!